docs/source/array-numpy-compatibility.rst

```diff
@@ -0,0 +1,377 @@
+Compatibility with numpy functions
+==================================
+
+The following table describes the compatibilities between `numpy` and `dask.array`
+functions.
+Please be aware that some inconsistencies with the two versions may exist.
```

Comment:
Some things I might add here:

- A note that this table was/is updated manually and may not reflect the state of current Dask
- Brief explanation of what certain things represent within the table, i.e. what Dask equivalent vs. direct implementation entails, just confirming the fact that a blank entry indicates that the function isn't implemented in Dask

Reply:
Thanks @charlesbluca! I have added what you suggested.
Also, note that digging in the issue tracker I found #3688: the repository https://github.com/data-apis/array-api-comparison is cited there and I believe it is a relevant resource to be linked in this page.